scripts.js

```diff
@@ -66,9 +60,9 @@ const addItem = function() {
   contentContainer.innerHTML = addMove.outerHTML + addContent.outerHTML + addCopy.outerHTML + addDelete.outerHTML;
```

Comment:
So the way you're appending content here is a bit hacky, since at basic level you're just concatenating strings. 

_(Luckly for you, the modern browser knows how to interpret this, and will parse them correctly. This would have not work in the olden days)_

This is why your events listeners aren't being added!

A better approach is to use the `appendChild` method.

Eg:
```js
  contentContainer.appendChild(addMove);
  contentContainer.appendChild(addContent);
  contentContainer.appendChild(addCopy);
  contentContainer.appendChild(addDelete);
```
This will ensure all the elements are added as Nodes and not as strigs